mis_builder_budget/views/mis_budget_by_account_item.xml

```diff
@@ -48,5 +48,6 @@
         <field name="view_mode">list,form</field>
         <field name="domain">[('budget_id', '=', active_id)]</field>
         <field name="context">{'default_budget_id': active_id}</field>
+        <field name="path">mis-budget-by-account-items</field>
```

Comment:
I'm not sure, @sbidoul : is `mis-budget-by-account-items` correct, or it should be `mis-budgets-by-account-item`